docs/standards/universal-profile/04-lsp6-key-manager.md

```diff
@@ -191,19 +196,29 @@ You can find **more details about each permissions by clicking on the toggles be
 </details>
 
 <details>
-    <summary><code>CHANGEPERMISSIONS</code> - Allows changing of permissions of addresses (adding + removing)</summary>
+    <summary><code>CHANGEPERMISSIONS</code> - Allows changing existing permissions of addresses</summary>
     <p style={{marginBottom: '3%', marginTop: '2%', textAlign: 'center'}}>
         <b>value = </b><code>0x0000000000000000000000000000000000000000000000000000000000000002</code>
     </p>
-    <p>This permission allows an address to <b>grant + revoke</b> permissions for any specific address (including itself).</p>
+    <p>This permission allows to <b>edit permissions</b> of any address that has already some permissions set on the ERC725Account (including itself).</p>
+
+![CHANGE Permissions](./../../../static/img/standards/lsp6-change-permissions.jpeg)
+
+<p>Bear in mind that the behaviour of CHANGEPERMISSIONS slightly varies, depending on the new permissions value being set (see figure below).</p>
```

Comment:
```suggestion
<p>Bear in mind that the behaviour of `CHANGEPERMISSIONS` slightly varies, depending on the new permissions value being set (see figure below).</p>
```

Not sure if the MD syntax works in this HTML part but maybe put the "code" syntax to this word would be nice?